src/cmd.rs

```diff
@@ -563,6 +563,7 @@ pub fn cluster_health(cluster_handle: &Rados) -> RadosResult<ClusterHealth> {
     });
     let result = cluster_handle.ceph_mon_command_without_data(&cmd)?;
     if let Some(return_data) = result.0 {
+        let return_data = String::from_utf8_lossy(&return_data);
```

Comment:
Given that we are returning a Result anyways, would it be worth changing to `String::from_utf8(&return_data)?;`